images/edk2-artifact/build.sh

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# UEFI Revocation List File can be downloaded from https://uefi.org
+
+usage() {
+    cat <<EOF
+    Usage: $0 [OPTIONS]
+    Options:
+    
+    Set brranch:                        --branch (example: v2.1 2.3 stable2024)
+    Set repository name:                --repo-name (example: edk2 libvirt etc)
+    Show this help message and exit:    -h, --help
+EOF
+    exit 0
+}
+
+parse_args() {
+    while [[ $# -gt 0 ]]; do
+    case "$1" in
```

Comment:
Let's add a separate debug flag instead of unnecessary logging via echo
```suggestion
    case "$1" in
    --debug)
       set -x 
       shift
       ;;
```